### PR TITLE
Motor Rotation

### PR DESCRIPTION
**Pull request by vasu737et**
Here we are able to rotate the motor in clockwise and anti-clockwise direction.

### CODE REVIEW

File: src/blocks/motors/virtual-circuit.ts [hunk 1]
```diff
@@ -29,12 +29,11 @@ export const motorCreate: AfterComponentCreateHook<MotorState> = (
 
 export const motorUpdate: SyncComponent = (state: MotorState, motorEl) => {
   const directionText = state.direction.toString();
-
+  motorEl.findOne("#motor").node.style.animation=state.direction==MOTOR_DIRECTION.FORWARD? "rotate 5s linear infinite":"rotateAntiClockwise 5s linear infinite";
```
Comment (phptuts) [inline src/blocks/motors/virtual-circuit.ts:32]: See if you can make it go faster if the speed increases.  You should be able to do that by changing the 5s and using the state.

Comment (pranavpr2404) [reply src/blocks/motors/virtual-circuit.ts:32]: Yes I have competed this task

Comment (phptuts) [reply src/blocks/motors/virtual-circuit.ts:32]: right now the motor will go at the same speed visually regardless of the speed the user puts in the block.  Can you make it go faster if the user puts in a higher speed.

**Comment by pranavpr2404** [comment]
Yes, I have made changes so that it changes its speed.

File: src/blocks/motors/virtual-circuit.ts (hunk 1 above)
Comment (phptuts) [reply src/blocks/motors/virtual-circuit.ts:32]: The number that's controlling the speed of the CSS animation is the `5s`

**Comment by phptuts** [comment]
@pranavpr2404 If the speed of the motor increases, you will need to decrease the seconds in the animation.

**Comment by pranavpr2404** [comment]
Sir,
Can we please meet on tomorrow morning 9.00am IST.

**Review by phptuts — APPROVED** [review]

**Pull request merged**